added to dictionaries.md replacing key values (#1407)
Codecademy/docs: content/python/concepts/dictionaries/dictionaries.md

`nums_squared` will produce a result of: `{ 1: 1, 2: 4, 3: 9, 4: 16, 5: 25 }`

The following built-in methods listed below help with manipulating dictionaries and the objects within:

> ## Replacing an Entry in a current Dictionary

Reviewer: ```suggestion
## Replacing an Entry in a Existing Dictionary
```

> If a key needs to be updated in an existing Dictionary, use the same syntax for adding an entry into your dictionary  ('dictionary[new_key]=new_value'), but instead replace the 'new_key' arguement for an already 'existing_key' in your desired dictionary, 'dictionary[existing_key] = new_value'

Reviewer: ```suggestion
If a key needs to be updated in an existing dictionary, it uses the same syntax for adding an entry into a dictionary  ('dictionary[new_key]=new_value'), but instead the 'new_key' argument is replaced with an already 'existing_key' in the desired dictionary, 'dictionary[existing_key] = new_value'
```

```py
person_age = { "Mark": 55, "Shiela": 28, "Bryce": 24, "Jim": 41, "Eric": 33, "Ally": 23 }

#Looks like it's Mark's Brithday! We should update our dictionary to reflect his new age

person_age['Mark'] = 56
```

> now when you call the value the "Mark" Key from the person_age dictionary it should produce the result: 56

Reviewer: ```suggestion

Now when the value in the "Mark" key is called from the `person_age` dictionary it will produce the result 56.
```

`nums_squared` will produce a result of: `{ 1: 1, 2: 4, 3: 9, 4: 16, 5: 25 }`

The following built-in methods listed below help with manipulating dictionaries and the objects within:

## Replacing an Entry in an Existing Dictionary

If a key needs to be updated in an existing dictionary, it uses the same syntax for adding an entry into a dictionary (`dictionary[new_key] = new_value`), but instead, the `new_key` argument is replaced with an already existing key in the desired dictionary (`dictionary[existing_key] = new_value`).

```py
person_age = { "Mark": 55, "Shiela": 28, "Bryce": 24, "Jim": 41, "Eric": 33, "Ally": 23 }

> #Looks like it's Mark's Brithday! We should update our dictionary to reflect his new age
> 

Reviewer 2: ```suggestion
# Looks like it's Mark's Birthday! Let's update our dictionary to reflect his new age:
```
